migrations/2022-01-04-034553_add_hidden_column/up.sql

```diff
@@ -0,0 +1,13 @@
+alter table community add column hidden boolean default false;
```

Comment:
@dessalines I think this create table and alter statement looks like all the rest. Let me know if something else is wrong.

Reply:
Looks good.